src/extensions/renderer/base/arrow-shapes.js

```diff
@@ -191,6 +191,56 @@ BRp.registerArrowShapes = function(){
     }
   } );
 
+  defineArrowShape( 'triangle-cross', {
+    points: [
+      -0.15, -0.3,
+      0, 0,
+      0.15, -0.3,
+      -0.15, -0.3
+    ],
+
+    crossLinePoints: [
+      -0.24175, -0.4,
+      0.24175, -0.4,
+    ],
+
+    forceStroke: true,
+
+    matchEdgeWidth: true,
+
+    scaleCrossLineYCoord: function( size, edgeWidth ){
+      return -0.01 + ( edgeWidth * 0.012 ) + ( math.log2( size - 28.95 ) * 0.001 );
+    },
+    scaleCrossLineXCoord: function( size, edgeWidth ){
+      return 0.42 + ( edgeWidth * 0.012 ) +  ( math.log2( size - 28.95 ) * 0.001 );
+    },
```

Comment:
These two functions could be changed to just one, with the costant as a parameter, but I thought it would be more clear that both the x and y coordinates of the cross can be scaled independently.